### DISCUSSION TITLE
Gap Years 尾声 | WhaleVocal

### DISCUSSION THREAD

**Discussion by giscus** [Octobug/blog#23] [Announcements]
# Gap Years 尾声 | WhaleVocal

Octobug's blog.

https://blog.octobug.site/posts/ending-the-gap-years

**Comment by SilverRainZ** [comment]
一直断断续续在 Github 上看你更新笔记，很惊奇，也很佩服你在 GAP 期间选择做这样的事情。
功利地说，你读过的书刷过的题以及学会的英语，能在你接下来的工作里提供巨大的帮助。不过原谅我擅自加戏，最珍贵的事情也许是日拱一卒然后达成目标的成功经验，还有对自己更多的审视和了解。

**Reply by Octobug** [reply]
其实早期也浪费了不少时间，计划一直在微调；gap 这么久有一部分因素也是对以前某些时段过于松懈的补救，细想起来成本大得吓人。更健康的做法是寻求工作-生活-学习的平衡，不过要达成这种平衡本身也很难，如今要找到满足这个需要的工作非常不容易……

**Comment by Dinesht04** [comment]
"今天是愚人节。在这样的大环境里，主动 gap 确实算挺愚的，我愚我自己。" 😂😂
